.github/workflows/mkdocs.yaml

```diff
@@ -0,0 +1,58 @@
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
Let's enable this for all branches and PRs. We don't want new PRs breaking docs.

Reply:
Yup, good idea, I have enabled the build for each push but it will only deploy for main or master. Will make the change on the remaining branches too